soh/soh/Enhancements/randomizer/fishsanity.cpp

```diff
@@ -53,7 +53,7 @@ std::unordered_map<int8_t, RandomizerCheck> Rando::StaticData::randomizerGrottoF
 
 ActorFunc drawFishing = NULL;
 ActorFunc drawEnFish = NULL;
-Color_RGBA16 fsPulseColor = { 30, 240, 200 };
+Color_RGBA16 fsPulseColor = { 30, 16, 200 };
```

Comment:
Looking at code, it's 5:5:5:1 for bits, not sure what this is really supposed to be. If these colors were meant to be 8bit RGBA, it'd be `{4, 30, 25, 1}`

Reply:
Yeah `Color_RGBA16` is the wrong color type to use. Should be `Color_RGB8` or `Color_RGBA8`

Reply:
Updated to `Color_RGB8`